commit 4e1c695676bafbaecc9fb307f6ee94138ae72c12
Author: Jia Tan <jiat0218@gmail.com>
Date:   Wed Dec 20 22:19:19 2023 +0800

    Docs: Update --enable-sandbox option in INSTALL.
    
    xzdec now also uses the sandbox when its configured.

diff --git a/INSTALL b/INSTALL
index 57f89cce..d6eaa616 100644
--- a/INSTALL
+++ b/INSTALL
@@ -523,13 +523,16 @@ XZ Utils Installation
                 if any -fsanitize= option is specified in CFLAGS.
 
     --enable-sandbox=METHOD
-                There is limited sandboxing support in the xz tool. If
-                built with sandbox support, it's used automatically when
-                (de)compressing exactly one file to standard output and
-                the options --files or --files0 weren't used. This is a
-                common use case, for example, (de)compressing .tar.xz
-                files via GNU tar. The sandbox is also used for
-                single-file 'xz --test' or 'xz --list'.
+                There is limited sandboxing support in the xz and xzdec
+                tools. If built with sandbox support, xz uses it
+                automatically when (de)compressing exactly one file to
+                standard output when the options --files or --files0 aren't
+                used. This is a common use case, for example,
+                (de)compressing .tar.xz files via GNU tar. The sandbox is
+                also used for single-file 'xz --test' or 'xz --list'.
+                xzdec always uses the sandbox, except when more than one
+                file are decompressed. In this case it will enable the
+                sandbox for the last file that is decompressed.
 
                 Supported METHODs:
 